catalog/resource_sql_table.go

```diff
@@ -439,18 +439,14 @@ func columnChangesCustomizeDiff(d *schema.ResourceDiff) error {
 func ResourceSqlTable() common.Resource {
 	tableSchema := common.StructToSchema(SqlTableInfo{},
 		func(s map[string]*schema.Schema) map[string]*schema.Schema {
-			s["data_source_format"].DiffSuppressFunc = func(k, old, new string, d *schema.ResourceData) bool {
-				if new == "" {
```

Comment:
Hi @alexott, do you know why we return true if new is empty? 
`common.EqualFoldDiffSuppress` would return false if it's empty since it doesn't match with old string which should be the expected behaviour.

Reply:
it's optional field, so we may not specify it for managed tables, but backend will return DELTA for it (as I remember)

Reply:
I suggest that we leave it for `data_source_format`